lib/workflows/provision.js

```diff
@@ -14,6 +14,7 @@
 
 var async = require('async');
 var childProcess = require('child_process');
+var jsprim = require('jsprim');
```

Comment:
Is `jsprim` available to be `require`d as a module from a workflow job?

Reply:
nope. :(